src/logics/ArithLogic.h

```diff
@@ -170,10 +164,23 @@ class ArithLogic : public Logic {
     bool isIntNeg(SymRef sr) const { return sr == sym_Int_NEG; }
     bool isRealNeg(SymRef sr) const { return sr == sym_Real_NEG; }
 
-    bool isTimes(SymRef sr) const { return isIntTimes(sr) or isRealTimes(sr); }
+    bool isTimes(SymRef sr) const { return isTimesLin(sr) or isTimesNonlin(sr) or isTimesUnparsed(sr); };
+    bool isTimesLinNonlin(SymRef sr) const { return isTimesLin(sr) or isTimesNonlin(sr); };
+    bool isTimesLin(SymRef sr) const { return isIntTimesLin(sr) or isRealTimesLin(sr); }
+    bool isTimesUnparsed(SymRef sr) const { return isIntTimes(sr) or isRealTimes(sr); }
```

Comment:
What does it mean that the times is "unparsed"?

Reply:
That I understand, but the name does not convey this meaning.
Is this even used anywhere? I think the existence of this symbol should not be public information, no?

Reply:
You are right, this function is used in `isTimes` only

The problem here is that we don't have `private` in the `ArithLogic.h` at all, so I didn't want to brake this "singular style"...
This maybe should be another issue to work on the methods in `ArithLogic` in general to move part of them into the private zone?

Reply:
There is `protected` section.

Reply:
I removed the function...